fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -17,11 +17,16 @@
       # recreate/reload connections
       reload_connections false
       reload_on_failure false
-      flush_interval "#{ENV['ES_FLUSH_INTERVAL'] || '5s'}"
+      flush_interval "#{ENV['ES_FLUSH_INTERVAL'] || '1s'}"
       max_retry_wait "#{ENV['ES_RETRY_WAIT'] || '300'}"
       disable_retry_limit true
       buffer_type file
       buffer_path '/var/lib/fluentd/buffer-es-copy-config'
-      buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
-      buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT'] || '1m' }"
+      buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT'] || '32' }"
+      buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT'] || '8m' }"
+      # the systemd journald 0.0.8 input plugin will just throw away records if the buffer
```

Comment:
I think we can remove this comment entirely.

Reply:
@portante file a new issue - this PR is a cherry-pick of a cherry-pick of commits that have already been pushed